connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -59,6 +56,7 @@ import org.apache.spark.unsafe.types.CalendarInterval
 // scalastyle:on
 class PlanGenerationTestSuite
     extends ConnectFunSuite
+    with RemoteSparkSession
```

Comment:
The whole point of this test was not to use a remote session. Can we just move the test to sql to ClientE2ETestSuite?

Reply:
Done